meta-lxatac-software/recipes-rust/bottom/bottom_0.10.2.bb

```diff
@@ -1,4 +1,4 @@
-inherit cargo
+inherit cargo cargo-update-recipe-crates
```

Comment:
Could you specify the "now" from the commit message a bit more? I see the `bottom_${PV}` include existed before but was just a dummy. So in how did this change now? (I'm really not a yocto rust expert, yet)

Reply:
I assumed that `cargo-update-recipe-crates.bbclass` is a recent addition, but it is inded in poky since nanbield.
How the time flies. I have updated the commit message on why this is a good change (more convenient tool and integrated in poky, instead of external).